tools/update_repo.sh

```diff
@@ -19,6 +19,8 @@ aws="aws --endpoint-url ${AWS_S3_ENDPOINT_URL:-https://hb.bizmrg.com} s3"
 aws_cp_public="$aws cp --acl public-read"
 aws_sync_public="$aws sync --acl public-read"
 
+arch=$(rpm --eval '%{_arch}' || dpkg --print-architecture)
```

Comment:
It seems, I misguided you when I show example how to better determine a distro specific name of a current architecture:

> ```
> (centos:7) # rpm --eval '%{_arch}'
> x86_64
> (centos:7) # uname -m
> x86_64
>
> (ubuntu:focal) # dpkg --print-architecture
> amd64
> (ubuntu:focal) # uname -m
> x86_64
> ```
>
> https://superuser.com/a/106545/449779
> https://stackoverflow.com/questions/20988371/linux-bash-get-releasever-and-basearch-values#comment104803498_20990596

My bad, I just wanted to show that we have x86_64 and aarch64 on CentOS, Fedora and openSUSE Leap, while amd64 and arm64 on Debian and Ubuntu. Harcoding those names would be okay.

Reply:
I rewrote it.